pwnlib/rop/rop.py

```diff
@@ -379,13 +379,11 @@ class ROP:
     #: which is not contiguous
     migrated = False
 
-    def __init__(self, elfs, base=None, **kwargs):
+    def __init__(self, elfs, base=None, should_load_gadgets=True, **kwargs):
         """
         Arguments:
             elfs(list): List of ``pwnlib.elf.ELF`` objects for mining
         """
-        import ropgadget
```

Comment:
Also it looked like this wasn't needed here so I removed it. Motivation being that capstone can be annoying to install.